### PR TITLE
simplify hl_lines ranges and standardize 2 spaces between each range

### PR DESCRIPTION
**Pull request by la-mar**
Addresses issue #1859.

All changes fall into two categories:

1. Update all ranges to new compressed format as specified in the original issue.
2. Standardize on two spaces between individual ranges.  This was already incorporated in some places and felt easier to read.

### CODE REVIEW

**Review by txemac — APPROVED** [review]

File: docs/en/docs/advanced/testing-dependencies.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ To override a dependency for testing, you put as a key the original dependency (
 
 And then **FastAPI** will call that override instead of the original dependency.
 
-```Python hl_lines="26 27 30"
+```Python hl_lines="26-30"
```
Comment (txemac) [inline docs/en/docs/advanced/testing-dependencies.md:31]: This change should be `"26-27  30"`

**Review by txemac — CHANGES_REQUESTED** [review]

File: docs/en/docs/advanced/testing-dependencies.md (hunk 1 above)
Comment (la-mar) [reply docs/en/docs/advanced/testing-dependencies.md]: Good eye. Fixed it.

**Review by txemac — APPROVED** [review]

File: docs/en/docs/tutorial/query-params.md [hunk 2]
```diff
@@ -138,7 +138,7 @@ If you don't want to add a specific value but just make it optional, set the def
 
 But when you want to make a query parameter required, you can just not declare any default value:
 
-```Python hl_lines="6 7"
+```Python hl_lines="6  7"
```
Comment (JulianMaurin) [inline docs/en/docs/tutorial/query-params.md:141]:
```suggestion
```Python hl_lines="6-7"
```

File: docs/en/docs/advanced/testing-database.md [hunk 3]
```diff
@@ -48,7 +48,7 @@ For the tests we'll use a file `test.db` instead of `sql_app.db`.
 
 But the rest of the session code is more or less the same, we just copy it.
 
-```Python hl_lines="8  9  10 11 12 13"
+```Python hl_lines="8  9  10-13"
```
Comment (JulianMaurin) [inline docs/en/docs/advanced/testing-database.md:51]:
```suggestion
```Python hl_lines="8-13"
```

**Review by JulianMaurin — CHANGES_REQUESTED** [review]
Hello @la-mar 👋 

I've just spotted 2 small mistakes 👏
Also you miss `docs/en/docs/tutorial/middleware.md` (L.53):
from `Python hl_lines="10  12  13"` to `Python hl_lines="10  12-13"`

Thanks for your work

File: docs/en/docs/tutorial/query-params.md (hunk 2 above)
Comment (la-mar) [reply docs/en/docs/tutorial/query-params.md]: Thanks for spotting the oversights. Just fixed em!

**Review by ghost — APPROVED** [review]

**Review by JulianMaurin — APPROVED** [review]
Well done @la-mar 👏 
(wooops, previously approved with my business account 😬)

**Comment by tiangolo** [comment]
Ah, this is amazing @la-mar ! Beautiful job! And thanks for also updating the double spaces to separate blocks. :art: :rocket: :tada: 

And thanks @txemac and @JulianMaurin for the reviews! :bow: :coffee:

**Pull request merged**